// duracell/statements-routing.module.ts
import { NgModule }            from '@angular/core';
import { RouterModule, Routes }        from '@angular/router';

import { StatementsModule }    from './statements.module';
import { InputComponent }    from './input.component';
import { RecapComponent }    from './recap.component';
import { ConfirmationComponent }    from './confirmation.component';

/*
    ANGULAR2-NEXUS-STARTER NOTE:
    For Routing Modules related to modules with no sub modules follow the current example
*/
export const STATEMENTS_ROUTES: Routes = [
	/*
      	ANGULAR2-NEXUS-STARTER NOTE:
      	Redirect to the first page for the module. 
  	*/
  	{ 
  		path: '', 
  		redirectTo: 'input', 
  		pathMatch: 'full' 
  	},
  	/*
    	ANGULAR2-NEXUS-STARTER NOTE:
		Specify the path and the component for every page.
  	*/
  	{ 
  		path: 'input', 
  		component: InputComponent 
  	},
  	{ 
  		path: 'recap', 
  		component: RecapComponent 
  	},
  	{ 
  		path: 'confirmation', 
  		component: ConfirmationComponent 
  	}
];

@NgModule({
  	imports: [
  		RouterModule.forChild(STATEMENTS_ROUTES),
      StatementsModule
  	]
})
export class StatementsRoutingModule {}
